tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -1005,7 +1062,29 @@ function setupBabel(tree: Tree, options: NormalizedSchema) {
   pkgJson.devDependencies = pkgJson.devDependencies || {};
 
   const shouldAddGriffelPreset = pkgJson.dependencies['@griffel/react'] && packageType === 'web';
-  const extraPresets = shouldAddGriffelPreset ? ['@griffel'] : [];
+  const extraPresets = shouldAddGriffelPreset
+    ? [
```

Comment:
isn't there a better way how to define this in a common config ?

Reply:
Just tested in another branch and using the `extend` option for `.babelrc.json` files does work! Added a `.babelrc-v9.json` file to root of monorepo and updated the workspace generator to reflect this update in [3616186](https://github.com/microsoft/fluentui/pull/26527/commits/36161867412b1024d61b606d4c7164713373b275)